### PR TITLE
chore(appveyor): Add initial config

### PR DESCRIPTION
**Pull request by dignifiedquire**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 708c765f2c889a0458216a6dac0e19a984905a4a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at http://karma-runner.github.io/0.13/dev/git-commit-msg.html

---
This message was auto-generated by https://gitcop.com

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Test against this version of Node.js
+environment:
+  matrix:
+    # node.js
+    - nodejs_version: "0.10"
+    - nodejs_version: "0.12"
+    # io.js
+    - nodejs_version: "1.0"
```
Comment (mzgol) [inline appveyor.yml]: This should say `"4"` now (`"4"`, not "4.0")

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+# Install scripts. (runs after repo cloning)
+install:
+  # Get the latest stable version of Node.js or io.js
+  - ps: Install-Product node $env:nodejs_version
+  # Ensure the latest npm version
+  - npm install -g npm
```
Comment (mzgol) [inline appveyor.yml]: Does it really work on Windows/Appveyor? For me it was still picking up the included npm, not the installed one. Fixing that would most likely require messing with the `%PATH%` env variable.

I just settled to remove Node 0.10 from the setup in my packages; it's already tested via Travis and Node >=0.12 have fully updated npm@2 so they should work fine.

Comment (Dignifiedquire) [inline appveyor.yml]: At the moment a lot of this stuff is not working :( but you are right, we can probably get rid of this line

**Comment by dignifiedquire** [comment]
Will do this, some day. But today is not the day.

**Pull request closed without merging**